docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
         image: adminer
 
     vault:
-        image: vault:0.11.5
+        image: vault:1.0.3
```

Comment:
is this a requirement of the change? how does it affect the other functionality? do we need to update this in the chart as well?

Reply:
synced with @bonifaido: the PR should be merged only after the chart has been updated

Reply:
which is: https://github.com/banzaicloud/banzai-charts/pull/713

Reply:
It has been merged and deployed to all envs.

Reply:
Please update to 1.1.0 to have the same version across all projects.

Reply:
Done.